setup.py

```diff
@@ -22,7 +22,7 @@ def get_version(rel_path):
 
 
 install_requirements = [
-    "boto3>=1.3,<2.0",
+    "boto3>=1.18.32,<2.0",
```

Comment:
Why was the boto3 requirement updated?

Reply:
Nice Catch, I was testing out the new DisableRollback feature. Must have slipped through my commits. I will revert this.